CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ IMPROVEMENTS:
 * provider: add `skip_verify` configuration to skip TLS verification ([#319](https://github.com/hashicorp/terraform-provider-nomad/pull/319))
 * provider: update Go to 1.20.5 ([#334](https://github.com/hashicorp/terraform-provider-nomad/pull/334))
 * resource/nomad_acl_policy: add support for `job_acl` ([#314](https://github.com/hashicorp/terraform-provider-nomad/pull/314))
+* resource/nomad_job: add support to import existing jobs. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359)]
```

Comment:
And volumes!

Reply:
At first I thought we didn't need one, since the `nomad_csi_volume*` resources are new, but they actually replace the previous volume resource so it's probably a good idea to add a note for them as well 👍